README.md

```diff
@@ -6,218 +6,11 @@
 
 <img src="https://raw.githubusercontent.com/chanzuckerberg/cellxgene/master/docs/cellxgene-demo-1.gif" width="200" height="200" hspace="30"><img src="https://raw.githubusercontent.com/chanzuckerberg/cellxgene/master/docs/cellxgene-demo-2.gif" width="200" height="200" hspace="30"><img src="https://raw.githubusercontent.com/chanzuckerberg/cellxgene/master/docs/cellxgene-demo-3.gif" width="200" height="200" hspace="30">
```

Comment:
I realize we don't want too much duplication of documentation since it makes it difficult to update and know where to get info, however, I think we should have a bit more in the readme. I don't want to turn off users who are doing their first pass eval and want to know the key info without having to follow a chain of links.

Reply:
Can you help me understand why you prefer to keep the developer guide in the readme but the user guide in the docs? I naively would have assumed the opposite :)

Reply:
Sorry, let me rephrase.

Users should not have to go to the repo to find what they need. If users come to the repo, we should help them get to the docs ASAP. This is not necessarily true for developers.

We *can* expect developers to explore the repo to find what they need, so the README is an OK place to put the developer guide. However, since it bloats the README, I think there are better places to put it:
- CONTRIBUTING.md
- in the main docs

I think a good model here is Atom

Their "flight manual" (docs) has three distinct sections: getting started, using, hacking
https://flight-manual.atom.io/

The README takes a very light touch, basically acting as a landing page or table of contents for other resources:
https://github.com/atom/atom/blob/master/README.md

The contributing guide sets expectations for all types of contributions and links out to other resources where relevant.
https://github.com/atom/atom/blob/master/CONTRIBUTING.md

Reply:
When I go to a readme for a project I want to be able to get a feel for the project and be able to download and use it if I like what I see. I personally get upset when I have to follow a series of links to get to the core info. 

To that point, I think developer docs can live outside the main readme. If I want to develop on the project that means that I've already used it in some capacity and am willing to invest more time.

However I disagree with the statement "If users come to the repo, we should help them get to the docs ASAP" They should be able to make an informed decision based on the readme whether they want to invest more of their time in the project (clicking links, reading docs, learning how to use options, etc.) No matter where we intercept them, the user should be able to get info on what cellxgene does, why they should use it over other options, how to get started initially, and how to get more info.

Reply:
+1, @csweaver